lib/generators/active_outbox/model/model_generator.rb

```diff
@@ -42,6 +48,10 @@ def migration_version
       def table_name
         model_name.blank? ? 'outboxes' : "#{model_name}_outboxes"
       end
+
+      def aggregate_identifier_type
+        options['uuid'].present? ? ActiveOutbox::AdapterHelper.uuid_type : 'integer'
```

Comment:
why `integer` ? just in case rails ids are `bigint`s, not sure if it affects here or not